### PR TITLE
feedback

### PR DESCRIPTION
**Pull request by s2s-ai-challenge**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -41,7 +42,7 @@ and a continously updating leaderboard. For code examples and how to contribute,
 2. 10K CHF
 3. 5K CHF
 
-The 3rd prize is reserved for the top contribution from developing countries (see [Table C p.166](https://www.un.org/development/desa/dpad/wp-content/uploads/sites/45/WESP2020_Annex.pdf) for country list). If such a contribution is already among the top 2, any thrid contribution will get the 3rd prize.
+The 3rd prize is reserved for the top contribution from developing countries (see [Table C p.166](https://www.un.org/development/desa/dpad/wp-content/uploads/sites/45/WESP2020_Annex.pdf) for country list). If such a contribution is already among the Top 2, any thrid contribution will get the 3rd prize.
```
Comment (s2s-ai-challenge) [inline index.md:47]: is there a better list we could take? do want developing countries or least developed countries or not developed countries?

File: index.md [hunk 2]
```diff
@@ -154,11 +155,11 @@ Answered questions from the issue tracker will be transferred to the [FAQ](https
...
-- All taxes imposed on prizes are the sole responsibility of the winners.
-- By joining the competition (see steps https://renkulab.io/projects/aaron.spring/s2s-ai-challenge-template), participants agree that they will make their private repositories on renkulab.io public after the competition ends (31st October 2021) regardless whether their contributions are among the top 3 for prizes. All repositories must be made availbale under the xyz-tbd licence.
-- warranty?
-- law?
-- some WMO liability statement?
+- All taxes imposed on prizes are the sole responsibility of the winners. 
+- By joining the competition (see steps https://renkulab.io/projects/aaron.spring/s2s-ai-challenge-template), participants agree that they will make their private repositories on renkulab.io public after the competition ends (31st October 2021) regardless whether their contributions are among the top 3 for prizes. All repositories must be made availbale under the xyz-tbd licence. 
+- warranty? 
+- law? 
+- some WMO liability statement?
```
Comment (s2s-ai-challenge) [inline index.md:194]: @WenchaoCao do we need something from WMO here?

File: index.md [hunk 3]
```diff
@@ -154,11 +155,11 @@ Answered questions from the issue tracker will be transferred to the [FAQ](https
 - One team can only get one prize.
 - One Person can only join on team.
 - Model training is not allowed use ground truth/observations data after forecast was issued, see Data Timings.
-- All taxes imposed on prizes are the sole responsibility of the winners.
-- By joining the competition (see steps https://renkulab.io/projects/aaron.spring/s2s-ai-challenge-template), participants agree that they will make their private repositories on renkulab.io public after the competition ends (31st October 2021) regardless whether their contributions are among the top 3 for prizes. All repositories must be made availbale under the xyz-tbd licence.
-- warranty?
-- law?
-- some WMO liability statement?
+- All taxes imposed on prizes are the sole responsibility of the winners.
```
Comment (s2s-ai-challenge) [inline index.md:190]: taken from kaggle: https://www.kaggle.com/c/coleridgeinitiative-show-us-the-data/rules

File: index.md [hunk 4]
```diff
@@ -154,11 +155,11 @@ Answered questions from the issue tracker will be transferred to the [FAQ](https
 - One team can only get one prize.
 - One Person can only join on team.
 - Model training is not allowed use ground truth/observations data after forecast was issued, see Data Timings.
-- All taxes imposed on prizes are the sole responsibility of the winners.
-- By joining the competition (see steps https://renkulab.io/projects/aaron.spring/s2s-ai-challenge-template), participants agree that they will make their private repositories on renkulab.io public after the competition ends (31st October 2021) regardless whether their contributions are among the top 3 for prizes. All repositories must be made availbale under the xyz-tbd licence.
-- warranty?
-- law?
-- some WMO liability statement?
+- All taxes imposed on prizes are the sole responsibility of the winners. 
+- By joining the competition (see steps https://renkulab.io/projects/aaron.spring/s2s-ai-challenge-template), participants agree that they will make their private repositories on renkulab.io public after the competition ends (31st October 2021) regardless whether their contributions are among the top 3 for prizes. All repositories must be made availbale under the xyz-tbd licence.
```
Comment (s2s-ai-challenge) [inline index.md:191]: sounds reasonable? hard to effectively force participants

**Comment by FredericVitart** [comment]
Looks good. Just a few comments:
- Website will need to contain more text about the purpose of this competition (like the first paragraph of the flyer but a bit extended). original competition draft text could be used
- It should be clearer which output is expected: 1.5. degree grid weeks 3+4 and 5+6 forecasts, regions, output format
- It should be clear that people are not forced to use the datasets we will provide. they can use purely empirical methods
- It should clearer that people could join at any time before 1st November
- Will contestant be forced to used Renku? What if a contestant wants to do the whole calculation in his own system and just send us the code and forecasts in netcdf format?
- I think that the contestants should first register by sending an email or filling up a form. That should allow us to keep track of how many contestants we have
- Regarding ECMWF provision of CPU time, it may need to be mentioned that it is not sure this will be possible if the demand is too high
- The front page may be improved with a picture of 2 (maybe same as the one used in the flyer)
- I agree it would be nice to have a logo
- Leaderboard: I think it would be good to have 1 per parameter and 1 per region

Overall i think that we need to send the flyer around as soon as possible with the link to  he website. I don't think the website needs to be completely finalized by then, but at least contain the crucial informations. I think it is close to it.

**Comment by floriankrb** [comment]
in https://renkulab.io/projects/florian.pinault/s2s-ai-challenge-template : "Make the project private (so that other people do not steal your idea/code), etc.."
The image is a good idea. I spent a few minutes looking for all this in the renku project before realizing that I should look in the gitlab one.

**Comment by aaronspring** [comment]
> in https://renkulab.io/projects/florian.pinault/s2s-ai-challenge-template : "Make the project private (so that other people do not steal your idea/code), etc.."
> The image is a good idea. I spent a few minutes looking for all this in the renku project before realizing that I should look in the gitlab one.

fixed now: https://renkulab.io/projects/aaron.spring/s2s-ai-challenge-template

File: index.md [hunk 5]
```diff
@@ -41,9 +44,9 @@ and a continously updating leaderboard. For code examples and how to contribute,
 2. 10K CHF
 3. 5K CHF
 
-The 3rd prize is reserved for the top contribution from developing countries (see [Table C p.166](https://www.un.org/development/desa/dpad/wp-content/uploads/sites/45/WESP2020_Annex.pdf) for country list). If such a contribution is already among the top 2, any thrid contribution will get the 3rd prize.
+The 3rd prize is reserved for the top contribution from developing countries (see [Table C p.166](https://www.un.org/development/desa/dpad/wp-content/uploads/sites/45/WESP2020_Annex.pdf) for country list). If such a contribution is already among the Top 2, any thrid contribution will get the 3rd prize.
 
-The organizers thank WWRP for providing the money of the prize. missing some org here?
+The organizers thank WWRP for providing the money of the prize.
```
Comment (aaronspring) [inline index.md:49]: should we give credit to the organizations funding the prize? wasnt it two orgs?

**Pull request merged**